src/server/game/Entities/AreaTrigger/AreaTriggerTemplate.h

```diff
@@ -164,6 +164,8 @@ class AreaTriggerMiscTemplate
         TimeToTarget        = 0;
         TimeToTargetScale   = 0;
 
+        ExtraScaleCurve     = { 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 1.0000001f, 1.0f };
```

Comment:
Nope, ExtraScaleCurve6 is a float (example : [5] AREATRIGGER_EXTRA_SCALE_CURVE + 6: 1073741831/2.000002) with default value 1